napalm/iosxr.py

```diff
@@ -23,11 +23,12 @@
 
 class IOSXRDriver(NetworkDriver):
 
-    def __init__(self, hostname, username, password):
+    def __init__(self, hostname, username, password, timeout=60):
         self.hostname = hostname
         self.username = username
         self.password = password
-        self.device = IOSXR(hostname, username, password)
+        self.timeout = timeout
```

Comment:
I guess there is not really a reason to assign timeout to self here, as I am only using it right there